src/Employer.Web/Extensions/RequestExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace Esfa.Recruit.Employer.Web.Extensions
+{
+    public static class RequestExtensions
+    {
+        public static string GetRequestUrl(this HttpRequest request)
```

Comment:
Would GetRootUrl be a better name? Can't help thinking it should include the port too. Maybe return a Uri object?